client/my-sites/checkout/composite-checkout/hooks/use-prepare-products-for-cart.ts

```diff
@@ -313,13 +342,13 @@ function createRenewalItemToAddToCart( productAlias, productId, purchaseId, sele
 		{
 			meta,
 			product_slug: productSlug,
-			product_id: productId,
+			product_id: parseInt( String( productId ), 10 ),
```

Comment:
You shouldn't need the `String( ... )` as this happens "automattically" but it's not hurting anything

Reply:
I tried it but it complained that `parseInt` required a string, and `productId` might be a `number`